### DISCUSSION TITLE
change bitcoin core directory

### DISCUSSION THREAD

**Discussion by VlVEK** [ordinals/ord#1642] [Q&A]
installed bitcoin core on Ubuntu LTS using snap

ord unable to recognise RPC though Sparrow can

getting error: failed to connect to Bitcoin Core RPC at 127.0.0.1:8332/wallet/ord
because: I/O error: No such file or directory (os error 2)

unable to use command ord --bitcoin-data-dir <BITCOIN_DATA_DIR> to change directory

how to fix this?

**Comment by MaxflowO2** [comment]
compiled mine from *[here](https://github.com/bitcoin/bitcoin)* just remember to drop `git checkout 2b87e905857ba456db2cb3d0297a3102f17417be` the 24.x branch last commit
